src/app/pages/send-crypto-asset/forms/btc/btc-crypto-currency-send-form.tsx

```diff
@@ -21,23 +21,24 @@ import { RecipientField } from '../../components/recipient-field';
 import { SelectedAssetField } from '../../components/selected-asset-field';
 import { SendAllButton } from '../../components/send-all-button';
 import { createDefaultInitialFormValues } from '../../form-utils';
+import { useBtcFees } from '../../hooks/use-fees';
 import { btcAddressValidator } from '../../validators/recipient-validators';
 
 interface BitcoinCryptoCurrencySendFormProps {}
 export function BitcoinCryptoCurrencySendForm({}: BitcoinCryptoCurrencySendFormProps) {
+  const navigate = useNavigate();
   const currentAccountBtcAddress = useCurrentAccountBtcAddressState();
   const btcCryptoCurrencyAssetBalance =
     useBitcoinCryptoCurrencyAssetBalance(currentAccountBtcAddress);
-  // TODO: Replace hardcoded number here (200) with the tx byte length
-  const { data: btcFeeEstimations } = useBitcoinFeeEstimations(200);
-  const navigate = useNavigate();
+  const txFees = useBtcFees();
 
   logger.debug('btc balance', btcCryptoCurrencyAssetBalance);
-  logger.debug('btc fees', btcFeeEstimations);
+  logger.debug('btc fees', txFees);
 
   const initialValues = createDefaultInitialFormValues({
     memo: '',
-    fee: null,
+    fee: 0,
+    feeType: 0,
```

Comment:
🤷🏼‍♂️

Reply:
I believe `feeType` was to distinguish in analytics which fee set was used (api, simulated, capped, etc).

Reply:
I am wrong here, it is recording which fee estimate index is selected not the fee calculation type.

Reply:
Does it map 1:1 with the `FeesType` enum?

Reply:
It does appear to map 1:1.